packages/web/src/components/search/DataSearch.js

```diff
@@ -998,21 +1018,30 @@ DataSearch.defaultProps = {
 	showClear: false,
 	strictSelection: false,
 	searchOperators: false,
+	size: 20,
 };
 
-const mapStateToProps = (state, props) => ({
-	selectedValue:
-		(state.selectedValues[props.componentId]
-			&& state.selectedValues[props.componentId].value)
-		|| null,
-	suggestions: state.hits[props.componentId] && state.hits[props.componentId].hits,
-	themePreset: state.config.themePreset,
-	isLoading: state.isLoading[props.componentId] || false,
-	error: state.error[props.componentId],
-	analytics: state.analytics,
-	config: state.config,
-	headers: state.appbaseRef.headers,
-});
+const mapStateToProps = (state, props) => {
+	const getSuggestionHits = () => {
```

Comment:
Again for the performance reasons don't define the function here, move it to some util file.